fcsparser/tests/test_fcs_reader.py

```diff
@@ -256,6 +257,14 @@ def test_cytek_x_p5_data_segment(self):
         )
         _assert_data_segment('Cytek xP5', expected_values, num_rows=2)
 
+    def test_multi_data_set(self):
+        """Test reading a file with 4 data sets"""
+        fname = FILE_IDENTIFIER_TO_PATH['guava muse']
+        meta, df = parse_fcs(fname, data_set = 0)
```

Comment:
Are you able to add a unit-test to verify the first few records in the first and in the last df, so the test verifies that `data_set=0` vs. `data_set=3` does something?

Reply:
Hi @bpteague, I was trying to add a few lines to verify the content of `meta` and `df`

https://github.com/eyurtsev/fcsparser/pull/43/files#diff-841b40df492dad36f471fc70264fd520ed73d03fd92ca08173bc048aaf7c1d22R274

https://github.com/eyurtsev/fcsparser/pull/43/files#diff-841b40df492dad36f471fc70264fd520ed73d03fd92ca08173bc048aaf7c1d22R279

It looks like `data_set=1`, `data_set=2` and `data_set=3` return the same values for both `meta` and `df`. 

Do you know if this is an issue with the parsing logic or some peculiarity about this FCS file?